cicecore/cicedyn/general/ice_init.F90

```diff
@@ -2742,10 +2743,10 @@ subroutine init_state
 
       if (grid_ice == 'CD' .or. grid_ice == 'C') then
 
-         call grid_average_X2Y('S',uvel,'U',uvelN,'N')
-         call grid_average_X2Y('S',vvel,'U',vvelN,'N')
-         call grid_average_X2Y('S',uvel,'U',uvelE,'E')
-         call grid_average_X2Y('S',vvel,'U',vvelE,'E')
+         call grid_average_X2Y('A',uvel,'U',uvelN,'N')
+         call grid_average_X2Y('A',vvel,'U',vvelN,'N')
+         call grid_average_X2Y('A',uvel,'U',uvelE,'E')
```

Comment:
Why is this change in the averaging type needed?

Reply:
This is just for the initialization of boxslotcyl test. uvelE and vvelN are defined from the uvel, vvel values. uvel(78,79) is initialized to 0 because of the land mask (no slip) while uvel(78,78) is initialized to 2.27. As uvelE is interpolated from uvel, uvelE(78,79) should be 2.27/2 =1.135. This is obtained by changing the averaging type.

Reply:
Ah, thank you.